### PR TITLE
feat(#92): Add instructions on onBoard flow

### PR DESCRIPTION
**Pull request by antoniel**
Add one more step on onBoard flow saying to user to save their account keys with the JP video

### CODE REVIEW

**Comment by antoniel** [comment]
This is the current implementation of JP video on onBoarding, there's a button to restart the video and another to the next step of onboarding, the second one is only enabled when the video gets in the end.

photo_2021-08-10 14 57 17

File: src/components/onboard/suggestToUserToBackupTheirKeys.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		<View style={{ zIndex: z }}>
+			<Video
+				source={network}
+				resizeMode='cover'
+				ref={videoPlayer => videoRef.current = videoPlayer}
+				onEnd={() => setIsVideoFinished(true)}
+				style={{
+					height: SCREEN_HEIGHT
+				}}
+			>
```
Comment (lucasfloriani) [inline src/components/onboard/suggestToUserToBackupTheirKeys.tsx:31]: Self close the video component

File: src/components/onboard/suggestToUserToBackupTheirKeys.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+			</Video>
+			<View style={styles.buttonWrap}>
+				<Button
+					accessibilityLabel='onboard-name-button'
+					// loading={updating}
```
Comment (lucasfloriani) [inline src/components/onboard/suggestToUserToBackupTheirKeys.tsx:36]: Remove commented line

File: src/components/onboard/suggestToUserToBackupTheirKeys.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+					style={{ ...styles.button }}
+					w={150}
+					size='large'
+					color={theme.colors.primary}
+				>
+					<Typography color={'white'}>Next</Typography>
+				</Button>
+				<Button
+					accessibilityLabel='onboard-name-button'
+					// loading={updating}
```
Comment (lucasfloriani) [inline src/components/onboard/suggestToUserToBackupTheirKeys.tsx:48]: Remove commented line

File: src/components/onboard/suggestToUserToBackupTheirKeys.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+					style={{marginRight: 10}}
+					onPress={() => videoRef.current && videoRef.current.seek(0)}
+					disabled={false}
+					w={32}
+					color={theme.colors.primary}
+				>
+            		<Icon name='repeat' color={'white'} size={32} />
```
Comment (lucasfloriani) [inline src/components/onboard/suggestToUserToBackupTheirKeys.tsx:55]: Fix indentation

**Review by lucasfloriani — CHANGES_REQUESTED** [review]

File: src/components/onboard/suggestToUserToBackupTheirKeys.tsx (hunk 4 above)
Comment (lucasfloriani) [reply src/components/onboard/suggestToUserToBackupTheirKeys.tsx]: Fixed

File: src/components/onboard/suggestToUserToBackupTheirKeys.tsx (hunk 3 above)
Comment (lucasfloriani) [reply src/components/onboard/suggestToUserToBackupTheirKeys.tsx]: Fixed

File: src/components/onboard/suggestToUserToBackupTheirKeys.tsx (hunk 2 above)
Comment (lucasfloriani) [reply src/components/onboard/suggestToUserToBackupTheirKeys.tsx]: Fixed

File: src/components/onboard/suggestToUserToBackupTheirKeys.tsx (hunk 1 above)
Comment (lucasfloriani) [reply src/components/onboard/suggestToUserToBackupTheirKeys.tsx]: Fixed

**Review by lucasfloriani — APPROVED** [review]

**Pull request merged**